esti/system_test.go

```diff
@@ -126,8 +126,12 @@ func createRepository(ctx context.Context, t *testing.T, name string, repoStorag
 func deleteRepositoryIfAskedTo(ctx context.Context, repositoryName string) {
 	deleteRepositories := viper.GetBool("delete_repositories")
 	if deleteRepositories {
-		client.DeleteRepositoryWithResponse(ctx, repositoryName)
-		logger.WithField("repo", repositoryName).Info("Deleted repository")
+		_, err := client.DeleteRepositoryWithResponse(ctx, repositoryName)
+		if err != nil {
+			logger.WithError(err).WithField("repo", repositoryName).Error("Deleted repository")
```

Comment:
```suggestion
			logger.WithError(err).WithField("repo", repositoryName).Error("Deleting repository")
```

It might not have been deleted!

Reply:
Updated message and also checked the status code from the response